app/assets/stylesheets/views/_landing_page/map/leaflet.scss

```diff
@@ -0,0 +1,752 @@
+/* stylelint-disable */
```

Comment:
Can we not get leaflet's css from the included node module, or is this custom stuff?

Reply:
Yes, you're right we can. Have updated (there are a few overrides that are needed, I've moved them across to the main map sass file).